frontend/src/components/phones/onboarding/RealPhoneSetup.module.scss

```diff
@@ -274,3 +260,162 @@
+    --PhoneInputCountryFlag-borderColor--focus: var(--PhoneInput-color--focus);
+    --PhoneInputCountryFlag-backgroundColor--loading: rgba(0, 0, 0, 0.1);
+  }
+
+  .PhoneInputInput {
+    @include text-body-xl;
+    border-radius: $border-radius-md;
+    display: block;
+    padding: $spacing-sm;
+    background-color: $color-grey-10;
+    width: 100%;
+    /* The phone number input stretches to fill all empty space */
+    flex: 1;
+    /* The phone number input should shrink
+        to make room for the extension input */
+    min-width: 0;
+  }
+
+  .is-error > input {
+    border: 2px solid $color-red-60;
+    box-shadow: 0 0 4px 4px $color-red-30;
+  }
+
+  .PhoneInputCountryIcon {
+    width: calc(
+      var(--PhoneInputCountryFlag-height) *
+        var(--PhoneInputCountryFlag-aspectRatio)
+    );
+    height: var(--PhoneInputCountryFlag-height);
+  }
+
+  .PhoneInputCountryIcon--square {
+    width: var(--PhoneInputCountryFlag-height);
+  }
+
+  .PhoneInputCountryIcon--border {
+    /* Removed `background-color` because when an `<img/>` was still loading
+       it would show a dark gray rectangle. */
+    /* For some reason the `<img/>` is not stretched to 100% width and height
+       and sometime there can be seen white pixels of the background at top and bottom. */
+    background-color: var(--PhoneInputCountryFlag-backgroundColor--loading);
+    /* Border is added via `box-shadow` because `border` interferes with `width`/`height`. */
+    /* For some reason the `<img/>` is not stretched to 100% width and height
+       and sometime there can be seen white pixels of the background at top and bottom,
+       so an additional "inset" border is added. */
+    box-shadow: 0 0 0 var(--PhoneInputCountryFlag-borderWidth)
+        var(--PhoneInputCountryFlag-borderColor),
+      inset 0 0 0 var(--PhoneInputCountryFlag-borderWidth)
+        var(--PhoneInputCountryFlag-borderColor);
+  }
+
+  .PhoneInputCountryIconImg {
+    /* Fixes weird vertical space above the flag icon. */
+    /* https://gitlab.com/catamphetamine/react-phone-number-input/-/issues/7#note_348586559 */
+    display: block;
+    /* 3rd party <SVG/> flag icons won't stretch if they have `width` and `height`.
+       Also, if an <SVG/> icon's aspect ratio was different, it wouldn't fit too. */
+    width: 100%;
+    height: 100%;
+  }
+
+  .PhoneInputInternationalIconPhone {
+    opacity: var(--PhoneInputInternationalIconPhone-opacity);
+  }
+
+  .PhoneInputInternationalIconGlobe {
+    opacity: var(--PhoneInputInternationalIconGlobe-opacity);
+  }
+
+  /* Styling native country `<select/>`. */
+
+  .PhoneInputCountry {
+    position: relative;
+    align-self: stretch;
+    display: flex;
+    align-items: center;
+    margin-right: var(--PhoneInputCountrySelect-marginRight);
+  }
+
+  .PhoneInputCountrySelect {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: 1;
```

Comment:
@Vinnl as I mentioned above: this CSS comes from a 3rd party package we are using. We can't modify the CSS classes directly. Same issue we had with the Toast component since we're using CSS modules. I didn't write the CSS inside the `:global` - do you have a better idea on how we want to pull it from the package to use it here?

Reply:
Yeah I guessed something like that later - see https://github.com/mozilla/fx-private-relay/pull/2385#discussion_r953681101.

Reply:
done - pretty sure I missed that - feels a lot cleaner, at least now we don't have to see it in our styling, lol.